Jenkinsfiles/release_pieline/Jenkinsfile

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2019 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+pipeline { 
+  agent none
+  parameters {
+    string(
+      name: 'BRANCH',
+      defaultValue:"master",
+      description: "Kata Containers Branch"
+    )
+    string(
+      name: 'NEW_VERSION',
+      defaultValue:"",
+      description: "Kata Containers version"
+    )
+  }
+  environment {
+    PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/usr/local/go/bin"
+    BRANCH="${params.BRANCH}"
+    NEW_VERSION="${params.NEW_VERSION}"
+  }
+  stages {
+    stage('Bump repos') {
+    // FIXME need better label for ubuntu, generic
+      agent { label 'ubuntu-1710' }
+      steps {
+      // Todo
+        withCredentials([string(credentialsId: 'TODO_NEED_CREDENTIAL_ID', variable: 'GITHUB_TOKEN')]) {
+          sh '''
+          set +x
+          #This is needed to setup github credentials to do push
+	  # We dont want to log this so send it to null
+          (
+        	set +x
```

Comment:
Can a `JenkinsFile` reference an external script? I think that would be clearer if possible.